# NB: there's a known bug when frames of exited functions are displayed
x = (i for i in range(10))

for e in x:
    print(e)
